### ISSUE TITLE
Problem with file and folder names containing spaces

### ISSUE THREAD

**Issue by ndward** [ev3dev/vscode-ev3dev-browser#62]
On my Windows 10 machine and using the July 9 2018 Stretch image (and all previous Stretch images) I can't run from VS Code any Python script whose file name contains spaces and neither can I run any Python script that is in a folder whose name contains spaces. For example, when I try to run (F5) a script called motorAtest.py in an open folder called **Demo scripts space** I get this error:

```
Starting: brickrun --directory=/home/robot/**Demo scripts space** /home/robot/**Demo scripts space**/motorAtest.py
Started.
----------
Starting remote process failed: Failed to change to directory '/home/robot/**Demo**' (No such file or directory)
----------
Exited with error code 1.
```

And when I try to run a file called **motorA test.py** in a folder whose name has no spaces I get:

```
Starting: brickrun --directory=/home/robot/Demo_scripts /home/robot/Demo_scripts/motorA test.py
Started.
----------
Starting remote process failed: Failed to execute child process "/home/robot/Demo_scripts/**motorA**" (No such file or directory)
----------
Exited with error code 1.
```

I get the same errors if I right-click the file in the EV3 Device Browser and choose 'Run'.

However, these scripts WILL run correctly if launched from Brickman.

**Comment by WasabiFan** [comment]
I believe this was fixed by #54 but [never released](https://github.com/ev3dev/vscode-ev3dev-browser/releases). Publishing a new release from `master` would probably fix the issue.

**Comment by dlech** [comment]
Oh man, I didn't realize we had stuff that wans't released. Should be fixed in v0.8.1, just published.
